CS4125Project/src/Database/LocalizationDB.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Database;
+
+import Language.LocalizationLanguage;
+import Stock.StockItem;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ *
+ * @author hello
+ */
+public class LocalizationDB implements IDatabase {
+    
+    private String filename;
+    private int currentLanguage = 0;
+    private LocalizationLanguage[] languages;
+    private HashMap<String, ArrayList<String>> localizations;
+    
+    /**
+     * A constructor sets filename and a hashmap of Localization
```

Comment:
could be simplified to:

` /* A constructor sets filename and a hashmap of Localization */`